smarts/core/vehicle_index.py

```diff
@@ -558,7 +558,7 @@ def _switch_control_to_agent_recreate(
 
         # Reserve space inside the traffic sim
         for traffic_sim in sim.traffic_sims:
-            if traffic_sim.manages_vehicle(vehicle.id):
+            if traffic_sim.manages_actor(vehicle.id):
```

Comment:
This now seems slightly odd, `vehicle.id` might work but it is not specifically the actor id.